uint32 user_id      # The user ID
duration wait_time  # Estimated wait time
duration time_left  # Time left for active user
